test/index.js

```diff
@@ -0,0 +1,164 @@
+'use strict';
+
+const should = require('chai').should(); // eslint-disable-line
```

Comment:
you can just `require('chai').shoud();` then you don't need eslint-disable and `globals` in eslintrc